common/middleware/locals.js

```diff
@@ -20,8 +17,11 @@ module.exports = function setLocals(req, res, next) {
     getLocal: key => res.locals[key],
     getMessages: () => req.flash(),
     canAccess: permission => {
-      const userPermissions = get(req.session, 'user.permissions')
-      return check(permission, userPermissions)
+      if (!req.checkPermissions) {
```

Comment:
What about using the existing name (`canAccess`) for the request method? I quite like that a `can...` nicely suggests the return of a boolean.

Reply:
Done